source/_integrations/counter.markdown

```diff
@@ -18,7 +18,7 @@ The `counter` integration allows one to count occurrences fired by automations.
 
 The preferred way to configure counter helpers is via the user interface. To add one, go to
 **{% my helpers title="Settings -> Devices & Services -> Helpers" %}** and click the add button;
-next choose the "**Counter**" option.
+next choose the "**{% my config_flow_start domain=counter title="Counter" %}**" option.
```

Comment:
```suggestion
next choose the **{% my config_flow_start domain=counter title="Counter" %}** option.
```
While you are at it, let's remove these. The text is already bold, highlighted in blue,  and underlined. (I know you did not put them there)